package-lock.json

```diff
@@ -3437,22 +3437,611 @@
 			}
 		},
 		"@wordpress/interface": {
-			"version": "1.0.6",
-			"resolved": "https://registry.npmjs.org/@wordpress/interface/-/interface-1.0.6.tgz",
-			"integrity": "sha512-sd9DnC8tHb4kuNAZEoSds7VUO50HrB2NahLlQ7dkN26bKCYibAMWUM5Jio1OhZKG8gYPG0jcWVPmnMeIYfzjjQ==",
+			"version": "1.1.0",
```

Comment:
This is concerning, should we pin it to the older version that was used with WP 5.7 to avoid including a copy of most of WP dependencies?

Reply:
I fixed it by pinning `@wordpress/interface` to `1.0.6`.